### DISCUSSION TITLE
Continuous training of the model.

### DISCUSSION THREAD

**Discussion by hamezksm** [deepgram/community#628] [Feature Requests]
I had a request to continuously train the model more. I speak English but I have realised that the model has an issue trying to comprehend my accent. I have to speak with a different accent so that it works better.

Aside from accents, the version of how to use deepgram API on flutter apps has some issues of the listeners still running after stop recording. Requesting to do a PR.

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jkroll-deepgram** [comment]
Hi @hamezksm, I'm sorry that you're finding that our models don't recognize your accent well. We are continuously re-training and improving our models on a wide variety of dialects and accents. I do recommend trying a few of our models, including `model=nova-2`, `model=nova`, and `model=enhanced`, to see which works best for your use case.

We welcome any PRs to our open-source repos!
